docs/static/monitoring-apis.asciidoc

```diff
@@ -65,6 +65,110 @@ being consumed by a monitoring tool, rather than intended for human
 consumption.  The default for the `human` flag is
 `false`.
 
+[[node-info-api]]
+=== Node Info API
+
+coming[5.0.0-alpha4]
+
+experimental[]
+
+The node info API retrieves information about the node.
+
+[source,js]
+--------------------------------------------------
+GET /_node/<types>
+--------------------------------------------------
+
+Where `<types>` is optional and specifies the types of node info you want to return.
+
+You can limit the info that's returned by combining any of the following types: 
+
+//REVIEWERS: It looks like you cannot combine types in a comma-spearated list like you can for the Elasticsearch cluster APIs. Is this intentional/correct behavior? If so, I need to change this to say "<type>".
```

Comment:
@suyograo I noticed this problem on a couple of APIs. Based on your response, I'm assuming this is a known bug. Let me know if I need to open an issue to track this.